plugins/woocommerce/src/Admin/API/Reports/Orders/DataStore.php

```diff
@@ -353,6 +353,11 @@ protected function include_extended_info( &$orders_data, $query_args ) {
 
 			if ( $is_variation ) {
 				$variation = wc_get_product( $product_data['id'] );
+				/**
+				 * Used to determine the separator for products and their variations titles.
+				 *
+				 * @since 6.6.0
```

Comment:
I think at that point it was part of the WooCommerce Admin package so it's a little bit of a gray area.  In my opinion these are only part of WooCommerce itself post-merge, so this should probably be `6.4.0` if that was the merge version.

I have not developed a strong opinion on this though, so if you feel strongly still I can update to `4.0.0`.

Reply:
True, I think my only point against that would be that the filter name hasn't changed. Meaning that any dev that was using it before the merge and looking at the documentation now might get confused.
Given our documentation for filters is near non-existent I don't think this will be a huge deal if it say's 6.4.0 or 4.0.0. But it might cause some 3PD head scratches.
So my preference would still be `4.0.0`, but I won't reject this PR because of that.